### DISCUSSION TITLE
Error 500

### DISCUSSION THREAD

**Discussion by MachadoSergio** [dompdf/dompdf#2520] [Show and tell]
Hi, i have a error 500.

 [message] => Uncaught Error: Class 'Cpdf' not found in /vagrant/vendor/dompdf/dompdf/src/Adapter/CPDF.php:198#012Stack trace:#012#0 /vagrant/vendor/dompdf/dompdf/src/CanvasFactory.php(57): Dompdf\Adapter\CPDF->__construct('letter', 'portrait', Object(Dompdf\Dompdf))#012#1 /vagrant/vendor/dompdf/dompdf/src/Dompdf.php(299): Dompdf\CanvasFactory::get_instance(Object(Dompdf\Dompdf), 'letter', 'portrait')#012#2 /vagrant/src/libs/Frog/Pdf.php(69): Dompdf\Dompdf->__construct(Object(Dompdf\Options))#012#3 

Your code you make that. 
$this->_pdf = new \Cpdf(
            $size,
            true,
            $dompdf->getOptions()->getFontCache(),
            $dompdf->getOptions()->getTempDir()
        );

**Comment by bsweeney** [comment]
How are you installing Dompdf? And how are you referencing it in your code? The Composer autoloader or the one provided as part of the packaged download both reference the Cpdf class under dompdf/lib.
